Assignments/stu1/data-modeling.sql

```diff
@@ -1,7 +1,44 @@
 /**
  * Ex. 11
  */
+BEGIN;
 
+select * from lemonade;
+select * from size;
+
+insert into size (lemonade_size, lemonade_price) values ('Small', '$1.00');
+insert into size (lemonade_size, lemonade_price) values ('Medium', '$2.00');
+insert into size (lemonade_size, lemonade_price) values ('Large', '$3.00');
+
+insert into lemonade (lemonade_flavor) values ('Lemon');
+insert into lemonade (lemonade_flavor) values ('Strawberry');
+
+select count(*) from cat;
+
+CREATE TABLE IF NOT EXISTS public.lemonade
```

Comment:
I think we need a day or date field on this table.  I'm not sure how this model supports keeping track of daily lemonade sales.

Reply:
Unless I'm missing something...

Reply:
I had this select count(lemonade_size_id) from lemonade; but I forgot to change it. I am having a hard time trying to get it to work.